app/services/katello/repository_type_manager.rb

```diff
@@ -0,0 +1,59 @@
+module Katello
+  class RepositoryTypeManager
```

Comment:
Also have a feeling @ehelms is not satisfied with RepositoryTypesManager spewing instances of himself. His comments made me think he'd prefer RepositoryTypeManager holding instances of RepositoryType objects. I wonder if there is a middle ground that will satisfy that ...